### PR TITLE
Add setting to hide the "Create" button in DAO Browser

### PR DESCRIPTION
**Pull request by TW80000**

### CODE REVIEW

File: src/foam/comics/v2/DAOControllerConfig.js [hunk 1]
```diff
@@ -112,6 +112,12 @@ foam.CLASS({
         // for it which doesn't jive with the AbstractFObjectPropertyInfo.
         return { class: 'foam.u2.borders.NullBorder' };
       }
+    },
+    {
+      class: 'Boolean',
```
Comment (jlhughes) [inline src/foam/comics/v2/DAOControllerConfig.js:117]: Why would we not make this permission based?

Comment (TW80000) [reply src/foam/comics/v2/DAOControllerConfig.js:117]: We could, but not all DAOs/models use permissions for authorization.

Comment (jlhughes) [reply src/foam/comics/v2/DAOControllerConfig.js:117]: Are we planning to copy all the comics/DAOController enableFoo properties?

**Review by jlhughes — APPROVED** [review]

File: src/foam/comics/v2/DAOControllerConfig.js (hunk 1 above)
Comment (TW80000) [reply src/foam/comics/v2/DAOControllerConfig.js:117]: We could. I wasn't planning on doing so until I had a concrete need though.

**Pull request merged**